setup.py

```diff
@@ -23,10 +23,12 @@
 with open(path.join(here, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
 
+# TODO: update cylc/cylc-flow dependency after cylc-flow PR has been merged
 install_requires = [
     'jupyterhub==1.0.*',
     'tornado==6.0.*',
-    'graphene-tornado==2.1.*'
+    'graphene-tornado==2.1.*',
+    'cylc-flow==8.*'
```

Comment:
Sorry David, @oliver-sanders had a good idea to actually make `cylc-uiserver` to always point to `cylc-flow`'s `master` branch. Could we change it back?

Reply:
I think it needs to be 

```python
    ('cylc @ https://github.com/cylc/cylc-flow'
     '/tarball/master#egg=cylc-8.0a1.dev')
```

The egg part with the version does not really matter much

Reply:
Install fails
```
Collecting cylc@ https://github.com/cylc/cylc-flow/tarball/master#egg=cylc_flow-8.0a1.dev from https://github.com/cylc/cylc-flow/tarball/master#egg=cylc_flow-8.0a1.dev (from cylc-uiserver==1.0)
  Downloading https://github.com/cylc/cylc-flow/tarball/master (699kB)
     |████████████████████████████████| 706kB 329kB/s 
  WARNING: Generating metadata for package cylc produced metadata for project name cylc-flow. Fix your #egg=cylc fragments.
```
```
ERROR: cylc-uiserver 1.0 requires cylc@ https://github.com/cylc/cylc-flow/tarball/master#egg=cylc_flow-8.0a1.dev, which is not installed.
```

Reply:
I think maybe

```
('cylc @ https://github.com/cylc/cylc-flow'
     '/tarball/master#egg=cylc-flow-8.0a1.dev')
```

The egg name was cylc... I think it should work if we just add the cylc-flow part?

Reply:
The following seems to work:
```
('cylc-flow @ https://github.com/cylc/cylc-flow'
     '/tarball/master#egg=cylc-8.0a1.dev')
```